Com.OneSignal.Android/Com.OneSignal.Android.csproj

```diff
@@ -40,6 +40,21 @@
     <Reference Include="System.Xml" />
     <Reference Include="System.Core" />
     <Reference Include="Mono.Android" />
+    <Reference Include="OneSignal.Android.Binding">
+      <HintPath>..\OneSignal.Android.Binding\bin\Release\OneSignal.Android.Binding.dll</HintPath>
+    </Reference>
+    <Reference Include="OneSignal.Android.Binding">
+      <HintPath>..\OneSignal.Android.Binding\bin\Release\OneSignal.Android.Binding.dll</HintPath>
+    </Reference>
+    <Reference Include="OneSignal.Android.Binding">
+      <HintPath>..\OneSignal.Android.Binding\bin\Release\OneSignal.Android.Binding.dll</HintPath>
+    </Reference>
+    <Reference Include="OneSignal.Android.Binding">
+      <HintPath>..\OneSignal.Android.Binding\bin\Release\OneSignal.Android.Binding.dll</HintPath>
+    </Reference>
+    <Reference Include="OneSignal.Android.Binding">
```

Comment:
Looks like VS just duplicated this? Looks like we can remove all but one of these.